examples/tutorials/regions.py

```diff
@@ -96,9 +96,8 @@
 fig.coast(
     region="d",
     projection="Cyl_stere/12c",
-    land="lightgray",
+    land="darkgray",
```

Comment:
The changes of `regions.py` don't help reduce the time. Could you revert the changes of `regions.py` in this PR? We can explore how to make it faster later.

Reply:
Updated in dfc3dee; I re-added the borders argument